android/src/com/frostwire/android/util/ImageLoader.java

```diff
@@ -148,6 +149,20 @@ private ImageLoader(Context context) {
         picasso.setIndicatorsEnabled(false);
     }
 
+    public void loadAndTransform(Uri uri, Transformation transformation, ImageView imageView) {
+        picasso.load(uri).fit().transform(transformation).into(imageView);
+    }
+
+    public void loadAndTransformWithAlternative(final Uri primaryUri, final Uri secondaryUri, final Transformation transformation, final ImageView mPhoto) {
+        picasso.load(primaryUri).fit().transform(transformation).into(mPhoto, new Callback.EmptyCallback() {
+            @Override
+            public void onError() {
+                loadAndTransform(secondaryUri, transformation, mPhoto);
+            }
+        });
+    }
+
```

Comment:
I'd refactor to avoid repeating the core logic with picasso:

```java
public void loadAndTransform(final Uri primaryUri, final Uri secondaryUri, final Transformation transformation, final ImageView imageView) {
    Callback.EmptyCallback callback = new Callback.EmptyCallback() {
        @Override
        public void onError() {
            if (secondaryUri != null) {
                loadAndTransform(secondaryUri, transformation, imageView);
            }
        }
    };
    picasso.load(uri).fit().transform(transformation).into(imageView, callBack);
}


public void loadAndTransform(Uri uri, Transformation transformation, ImageView imageView) {
    loadAndTransform(uri, null, transformation, imageView);
}

```